pingpong/ud_pingpong.c

```diff
@@ -519,7 +519,11 @@ static int run(void)
 		for (i = 0; i < TEST_CNT; i++) {
 			if (test_size[i].option > opts.size_option)
 				continue;
+
 			opts.transfer_size = test_size[i].size;
+			if (opts.transfer_size > buffer_size)
+				continue;
```

Comment:
That's what I was asking in the second half of my question, not sure how to handle that. I'd like symmetrical behavior, but I think it makes more sense to skip the ones larger than MTU in the predefined test cases and give a message in the case of a size given with `-S`.